pkg/resource/resource.go

```diff
@@ -632,8 +462,194 @@ func sendNodesMetrics(obj *unstructured.Unstructured, crName string) {
 
 	if len(nodesNames) != 0 {
 		nodesStr := strings.Join(nodesNames, ",")
-		metricsClient.SetUsedNodes(crName, obj.GetName(), obj.GetNamespace(), kind, nodesStr)
+		c.metricsClient.SetUsedNodes(crName, obj.GetName(), obj.GetNamespace(), kind, nodesStr)
 	} else {
-		log.Info("No assigned nodes for found for UsedNodes metric", "kind", kind, "name", obj.GetName(), "crName", crName)
+		c.log.Info("No assigned nodes for found for UsedNodes metric", "kind", kind, "name", obj.GetName(), "crName", crName)
+	}
+}
+
+func IsNamespaced(kind string) bool {
+	if kind == "Namespace" ||
+		kind == "ClusterRole" ||
+		kind == "ClusterRoleBinding" ||
+		kind == "SecurityContextConstraint" ||
+		kind == "SpecialResource" {
+		return false
+	}
+	return true
+}
+
+func IsNotUpdateable(kind string) bool {
+	// ServiceAccounts cannot be updated, maybe delete and create?
+	if kind == "ServiceAccount" || kind == "Pod" {
+		return true
+	}
+	return false
+}
+
+// Some resources need an updated resourceversion, during updates
+func NeedsResourceVersionUpdate(kind string) bool {
```

Comment:
lets also make it a private function

Reply:
We agreed to address this with https://issues.redhat.com/browse/MGMT-8567